spec/core-spec.md

```diff
@@ -189,7 +190,7 @@ Example:
 ```yaml
```

Comment:
Yes, the information is certainly redundant on a technical level. The reason I have included it in the simplified format is to try and reduce the "barrier to entry" of atomic for a new RT code using the format. In particular, for languages like C/C++ where unit support is poor and these things are harder to test, I like the idea of providing this stuff precomputed: it can provide a centralised implementation and reduce the surface area for bugs. Additionially it guarantees that all the calculations for these initial terms are done in double precision (considering things being ingested into GPU etc).

Whilst I am happy to reduce the number of things computed here, I am keen to keep them (and potentially add more), as the simplified format is primarily intended as a human-readable machine-interchange format, and the nature of using a YAML parser means a user can ignore any key they don't want. I don't expect any human to be writing models in the simplified format, as the higher level one offers a lot more flexibility.

Reply:
Ok, I misunderstood the purpose of the simplified format. I though it was meant for humans to write a quick model atom.

Reply:
Sorry I put  my comment in the wrong box... see below